### ISSUE TITLE
Items cant drop to first item

### ISSUE THREAD

**Issue by eskimoblood** [lukasoppermann/html5sortable#8]
I cant drag items into the first item in a column.

When I try to drop an item in a grid into the first place of a row, then it only works when I enter the item with the mouse from above. When the mouse comes from a the left side it does not work

**Comment by voidberg** [comment]
Sorry for replying so late.

Could you please create an example on [Plunkr](http://plnkr.co/) or [CodePen](http://codepen.io/)?

Thanks.

**Comment by sdettling** [comment]
I'm able to reproduce this as well in your examples. While dragging from a lower ordered item to a higher ordered item it works as expected, but when dragging from higher to lower the next sibling shows the placeholder, and will drop into that slot rather than the item you're hovering over.

Looks like it might have something to do with the "not" method. When I change line 123 to placeholders[0].detach(); it seems to work as expected but I haven't explored the implications of that change.

**Comment by voidberg** [comment]
This is very odd. Running the example linked from the Github repo exhibits this issue but running the same example locally does not. Will investigate.

**Comment by alpracka** [comment]
Did you solve it? In your examples I can't move any item over Item 1.

![Example](http://n1dev.com/system/img/sortable_bug.png)

**Comment by voidberg** [comment]
I didn't solve it. I can only reproduce it when I am running the example linked on Github. When ran locally everythings works fine.

If anyone has any ideas please share.

**Comment by Black-C3PO** [comment]
Couldn't you just update the example linked on Gitub.

The example in the branch works fine for me, too.

**Comment by denvil** [comment]
Hi. I managed to duplicate this issue. 
If sortable is run twice on list then this issue crops up. Still not sure what is duplicated but adding 

```
$(this).children(options.items).off('dragstart.h5s dragend.h5s selectstart.h5s dragover.h5s dragenter.h5s drop.h5s'); 
```

after line 25 fixed for me this issue.

In example this can be duplicated by adding second $('.sortable').sortable(); in example.

Not sure what the right fix is. Hope this helps.
